packages/conf-brotli/conf-brotli.0.0.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "fxfactorial"
+authors: "Edgar Aroutiounian"
+homepage: "https://github.com/google/brotli"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "BSD-3"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+depexts: [
+  [["debian"] ["libbrotli-dev"]]
+  [["freebsd"] ["libbrotli"]]
+  [["homebrew" "osx"] ["brotli"]]
+  [["ubuntu"] ["libbrotli-dev"]]
+]
```

Comment:
Could you please add an `install:` field which checks for the library via `pkg-config`. I.e. something like:
```
install: [ "pkg-config" "libbrotlidec" "libbrotlienc" ]
```